helper/helper-common/src/main/java/com/linkedin/avroutil1/compatibility/CodeGenerationConfig.java

```diff
@@ -54,18 +70,28 @@ public CodeGenerationConfig(
 
     @Deprecated
     public CodeGenerationConfig(StringRepresentation stringRepresentation) {
-        this(stringRepresentation, true, Collections.emptySet());
+        this(stringRepresentation, false, AvscGenerationConfig.VANILLA_ONELINE);
     }
 
     public StringRepresentation getStringRepresentation() {
         return stringRepresentation;
     }
 
+    public boolean isEnableAvro702Handling() {
```

Comment:
Nitpick... if you're going to rename this, then may I suggest `isAvro702HandlingEnabled()` which rolls off the tongue a bit more (well, as much as a JIRA ticket name can ever roll off the tongue ;D ...).